### DISCUSSION TITLE
forrtl: severe (157): Program Exception - access violation

### DISCUSSION THREAD

**Discussion by Sagar-Singhal** [firemodels/fds#13803] [Q&A]
Hi, 

I am trying to simulate 100 mm diameter circular pan, pool fire under crosswind of 1 ms-1. In this I am using elevated pan configuration to study the leeward and windward vortices and flame behaviour. But while running the simulation, I am getting an error "forrtl: severe (157): Program Exception - access violation". I am using "FDS-6.9.1-0-g889da6a-release" version, and windows operating system with CPU "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz", total number of processors are 28 divided amongst 7 meshes. 
Please see the attached .fds and error file.
[error.txt](https://github.com/user-attachments/files/17917965/error.txt)
[fds_code.txt](https://github.com/user-attachments/files/17917993/fds_code.txt)

**Comment by drjfloyd** [comment]
You have not defined a background species or initial mass fractions. Please review the FDS User's Guide for how to define species.
Since youe REAC is just the complete combustion of N-HEPTANE, I suggest you use the simple chemistry formulation of REAC and SPEC. Details are in the FDS User's Guide.
SOOT_YIELD has no meaning if simple chemistry is not being used.

**Reply by Sagar-Singhal** [reply]
Hi, I have tried changes as you mentioned, still I get same error. So, I thought to try some validation case, when I try running Phoenix fire validation case. I get same error.
[Phoenix_fire_error.txt](https://github.com/user-attachments/files/17921393/Phoenix_fire_error.txt)


Path : Validation/Phoenix_LNG_Fires/FDS_Input_Files/Phoenix02.fds

**Comment by drjfloyd** [comment]
From your first error.txt
 Number of MPI Processes:  7
 Number of OpenMP Threads: 4

The CPU you identified only has 18 physical cores and you are requesting 28. FDS doesn't tend to do well with hyperthreading. 

If I delete your SPEC lines and change your REAC and SURF to the inputs below, your first case started running and got by the 60 time steps where it died for you. I don't have 28 cores on my PC but ran as 1 MPI 1 Open MP thread, 7 MPI 1 Open MP thread and 1 MPI 4 OpenMP threads, and all ran beyond 60 time steps.  

&REAC ID='N-HEPTANE', FUEL='N-HEPTANE', SOOT_YIELD=0.015/

&SURF ID='FUEL'
      COLOR='RED'
      HRRPUA=400.0/

&SURF ID='WIND'
      COLOR='BLUE'
      VEL=-1./

**Reply by Sagar-Singhal** [reply]
I followed these changes but I got a numerical instability error at the time step of around 2120.

**Reply by drjfloyd** [reply]
It isn't clear what set of changes you made. Just the FDS inputs or did you also limit core usage after disabling hyperthreading?

I started this case on our linux cluster using the release version with 7 MPI processes and 4 OpenMP threads and it is out to over 17000 timesteps.

**Reply by Sagar-Singhal** [reply]
I did both, limiting the total number of core usage to 14 (7 MPI and 2 openmp) also. It gives error. I found that issue is with "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz" CPU. I also tried running simulation in two different machines, it is working successfully. Thank you very much for your kind help. I will try to figure out why segmentation error happening in machine.

**Comment by mcgratta** [comment]
This case ran with the latest nightly build. I suggest you try that.
